### PR TITLE
q 2.0.16

### PR DESCRIPTION
**Pull request by harelba**
- [ V] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ V] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ V] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ V] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ V] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Bump q to properly install 2.0.16 using venv, using @bayandin 's awesome help and recommendation.

I'm aware of the irrelevant broken symlink, will be removed in the next release and deleted from the formula.

Thanks @bayandin for the helpfulness and information!

### CODE REVIEW

File: Formula/q.rb [hunk 1]
```diff
@@ -1,16 +1,27 @@
+# frozen_string_literal: true
```
[the remark this answers is not in the document]
Comment (harelba) [reply Formula/q.rb:1]: `brew audit --strict` failed without this line on `C: 1: col 1: Missing frozen string literal comment.`. 

I'll push it anyway and see if everything passes.

File: Formula/q.rb [hunk 2]
```diff
@@ -1,16 +1,27 @@
...
+
+  resource "six" do
+    url "https://files.pythonhosted.org/packages/6b/34/415834bfdafca3c5f451532e8a8d9ba89a21c9743a0c59fbd0205c7f9426/six-1.15.0.tar.gz"
+    sha256 "30639c035cdb23534cd4aa2dd52c3bf48f06e5f4a941509c8bafd8ce11080259"
+  end
 
   def install
-    bin.install Dir["bin/*"] - ["bin/q.bat"]
-    bin.install_symlink bin/"q.py" => "q"
+    rm_f "bin/qtextasdata.py" # broken symlink
```
[the remark this answers is not in the document]
Comment (harelba) [reply Formula/q.rb:22]: sure.

File: Formula/q.rb [hunk 3]
```diff
@@ -1,16 +1,26 @@
+# q - Text as Data
```
Comment (bayandin) [inline Formula/q.rb:1]: Could you please remove this line:

```suggestion
```

And squash all the commits into a single one with a message `q 2.0.16`

Comment (harelba) [reply Formula/q.rb:1]: sure (same as the frozen string literals above - `brew audit --strict` failed on this missing comment, this is the reason i've added it in the first place).

**Review by bayandin — APPROVED** [review]

**Comment by bayandin** [comment]
Thank you @harelba 🎉

**Comment by harelba** [comment]
thank you @bayandin for helping, both me and my users 😄

**Pull request closed without merging**